docs/config/index.md

```diff
@@ -78,6 +78,8 @@ export default defineConfig(({ command, mode, ssrBuild }) => {
 })
 ```
 
+Only `ssrBuild` is included, as during dev, the config creates a single server shared between the SSR and non-SSR requests.
```

Comment:
I think we should move this paragraph below the next one as it's more common for users to use the `command` variable than `ssrBuild`.